src/nvim/os/env.c

```diff
@@ -215,31 +215,29 @@ void expand_env_esc(char_u *srcp, char_u *dst, int dstlen, bool esc, bool one,
   int c;
   char_u      *var;
   bool copy_char;
-  bool mustfree;                 /* var was allocated, need to free it later */
-  bool at_start = true;           /* at start of a name */
+  bool mustfree;  // var was allocated, need to free it later
```

Comment:
In this function, mustfree gets used a couple of different ways.  I started working on trying to clean that up, but in my opinion the changes required would've strayed too far from the purpose of this PR.  So I left mustfree in, and just set it when calling vim_getenv() to keep things consistent.